server/game/cards/03-WC/EDAIEdie4x4.js

```diff
@@ -0,0 +1,21 @@
+const Card = require('../../Card.js');
+
+class EdaiEdie4x4 extends Card {
+    setupCardAbilities(ability) {
+        this.play({
+            target: {
+                location: 'hand',
+                controller: 'self',
+                gameAction: ability.actions.archive()
+            }
+        });
+        this.persistentEffect({
+            targetController: 'opponent',
+            effect: ability.effects.modifyKeyCost(() => this.controller.archives.length || 0)
```

Comment:
I'm not sure what you were trying to do with the || 0, but this is redundant

Reply:
I copied that from another card, hell if I can remember which one.